src/github.js

```diff
@@ -48,6 +48,84 @@ const getRepos = async () => {
     }
   }
 
+  const getLanguages = async (repo) => {
+    try {
+      const { data } = await octokit.request('GET /repos/{owner}/{repo}/languages', {
+        owner: repo.full_name.split('/')[0],
+        repo: repo.name,
+      })
+      const languages = Object.keys(data); 
+      return languages
+    } catch (ex) {
+      throw ex
+    }    
+  }
+  
+  const getTeams = async (repo) => {
+    try {
+      const { data } = await octokit.request('GET /repos/{owner}/{repo}/teams', {
+        owner: repo.full_name.split('/')[0],
+        repo: repo.name,
+      })
+      const teams = data.map((t) => t.name);
+      return teams
+    } catch (ex) {
+      throw ex
+    }    
+  }
+
+ 
+  const getDotNetVersions = async (repo, keyword) => {
+    try {
+      const { data } = await octokit.request('GET https://api.github.com/search/code?q={keyword}+repo:{repo}', {
```

Comment:
Is there a reason we need to search vs know where this file is always located?

Reply:
Looking into that part (therefor I set the PR as Draft) - when Googling how to get all the csproj files in a GitHub repo the first answers are with search, but I think I have a way of using the Tree API and then get the content for the csproj files.